launcher.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+OLDDIR=`pwd`
+LIBDIR=/app/lib/poweriso
+export LD_LIBRARY_PATH=$LIBDIR
+export QT_QPA_PLATFORM_PLUGIN_PATH=$LIBDIR
+# export QT_DEBUG_PLUGINS=1
+cd $LIBDIR
+./poweriso
```

Comment:
`exec ./poweriso "$@"`

Reply:
It doesn't actually handle any command-line args.